packages/icons-react/types/Icon.d.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright IBM Corp. 2019, 2019
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import * as React from 'react';
+
+type IconSize = 16 | 20 | 24 | 32 | '16' | '20' | '24' | '32' | number | string;
```

Comment:
If this is changed to `... | '32' | (number & {}) | (string & {})` then autocomplete in IDE's will prefer the actual values rather than a generic string/number but still won't force it. See https://github.com/microsoft/TypeScript/issues/29729#issuecomment-505826972

Reply:
@joshblack Are there numbers outside of 16, 20, 24, and 32 that an Icon can use?